Extension/src/LanguageServer/client.ts

```diff
@@ -925,7 +923,6 @@ export class DefaultClient implements Client {
     public get AdditionalEnvironment(): { [key: string]: string | string[] } {
         return {
             workspaceFolderBasename: this.Name,
-            workspaceStorage: this.storagePath,
```

Comment:
Oh, does this regress user settings with ${workspaceStorage}? Or what uses this?

Reply:
If the TypeScript side needs the workspaceStorage and we want to computer it in cpptools, then this method could add a method to request it from cpptools.